test/bntest.c

```diff
@@ -10,6 +10,9 @@
 #include <errno.h>
 #include <stdio.h>
 #include <string.h>
+#ifdef __TANDEM
+#include <strings.h>
```

Comment:
What is this include required for? strcasecmp?

Reply:
Correct. For some historical reason, `strcasecmp` and related string functions of this form are in `strings.h` on this platform.

Reply:
Could you possibly put the dependency as a comment for these e.g
#include <strings.h> /* strcasecmp */
strcasecmp needs a cleanup at some point..
Some files are including "e_os.h", whilst others are doing this horrible #ifdef for windoze.

Reply:
Also the notation in openssl is to indent
e.g:
```
#ifdef X
# include Y
# ifdef
#  include
# endif
```

Reply:
Sure. Will fix these and others in the PR.